dagobah/core/core.py

```diff
@@ -117,12 +117,16 @@ def _add_job_from_spec(self, job_json, use_job_id=True):
             job.schedule(job_json['cron_schedule'])
 
         for task in job_json.get('tasks', []):
-            self.add_task_to_job(job,
-                                 str(task['command']),
-                                 str(task['name']),
-                                 soft_timeout=task.get('soft_timeout', 0),
-                                 hard_timeout=task.get('hard_timeout', 0),
-                                 hostname=task.get('hostname', None))
+            # If it is a jobtask, it will have a job_id
+            if task.get('job_id'):
```

Comment:
So, I wasn't sure what the best ordering or check would be for this. Simplest to me seemed to check for the existance of a job_id.